src/__tests__/__snapshots__/links.integration.test.js.snap

```diff
@@ -315,7 +315,10 @@ exports[`the build should not break any links 1`] = `
   "/guides/creating-test-money#on-this-page-title",
   "/guides/creating-test-money#via-api",
   "/guides/creating-test-money#via-centrapay-app",
-  "/guides/ecommerce-website",
```

Comment:
@daallen303 Change: Chuck a redirect in for this old url to avoid breaking this link

Reply:
I wasn't sure how to do a redirect so I just kept the filename the same. It's still a relevant title I was just trimming it to be a little neater.